docs/how-to/using-omnitrace-api.rst

```diff
@@ -159,6 +159,11 @@ Output from the API example program
 
 First, instrument and run the program.
 
+.. note::
+
+   For information on compiling and building Omnitrace, 
```

Comment:
Not quite. This document, <../reference/compiling-cmake.rst>, guides the user through linking the "omnitrace-user-library" to their own program. It is not how to build and install the tool, itself. 
These APIs are to allow the user to customize what they want to profile / trace by specifying their own "regions". This is what the example code above is doing. 
Now, I fear that other reads may have the same misunderstanding, and it may be best to bring the contents of "compiling-cmake.rst" back into this document, so that the connection is clear.

Reply:
Thanks for the clarification. The CMake document is relatively short, so I can certainly incorporate it here. I'll add some details from your explanation to give things more context.  I'll do this in my next update.

Reply:
I moved the content over and deleted the "compiling-cmake.rst" page. I agree that this makes things clearer. I restructured the document a bit, added some connecting information, and changed some of the headers.